src/string/isomorphism.rs

```diff
@@ -0,0 +1,59 @@
+/*
+Given two strings s and t, determine whether they are isomorphic.
+The two strings are isomorphic if the characters in s can be replaced by some mapping relation to get t
+*/
```

Comment:
I would suggest adding a file docstring here 
```suggestion
//! This module provides functionality to determine whether two strings are isomorphic.
//!
//! Two strings are considered isomorphic if the characters in one string can be replaced
//! by some mapping relation to obtain the other string.
```